web/src/styles/scss/save-button.scss

```diff
@@ -1,17 +1,23 @@
 @import '@cmsgov/design-system-support/src/settings/index';
 
-.save-button--container .ds-l-col--3 {
-  margin-left: 601px;
+.save-button--container {
+  max-width: inherit;
+  width: 100%;
```

Comment:
By inheriting the max-width and setting the width to 100%/margins to auto, we get a container that's the size of the `site-body` element and we can get rid of the grid system (since it won't really work here anyway).